eureka-core/src/main/java/com/netflix/eureka/registry/InstanceRegistry.java

```diff
@@ -80,7 +80,7 @@ boolean deleteStatusOverride(String appName, String id, InstanceStatus newStatus
 
     int getNumOfRenewsPerMinThreshold();
 
-    int isBelowRenewThresold();
+    int isBelowRenewThreshold();
```

Comment:
This cannot be renamed as it's a part of a public contract (an interface method of a public type).